web/rest/client/features/provider/recording/getRecording.feature

```diff
@@ -49,6 +49,8 @@ Feature: Retrieve recordings
               "fileSize": 4280,
               "mimeType": "audio/mpeg; charset=binary",
               "baseName": "7602fd7f-4153-4475-9100-d89ff70cdf76.0.mp3"
-          }
+          },
+          "ddi": null,
+          "user": null
```

Comment:
We should test that, these values can be updated, and this can be performed in the RecordingStub, having elements to null does not test if it is a relation to the required entities